contracts/libraries/PlasmaVaultLib.sol

```diff
@@ -126,9 +333,35 @@ library PlasmaVaultLib {
         emit ManagementFeeDataConfigured(feeAccount_, feeInPercentage_);
     }
 
-    /// @notice Gets the performance fee data
-    /// @return performanceFeeData The performance fee data, like the fee manager and the fee in percentage
-    //solhint-disable-next-line
+    /// @notice Gets the performance fee configuration data
+    /// @return performanceFeeData The current performance fee configuration containing:
+    ///         - feeAccount: Address receiving performance fees
```

Comment:
The address of the technical Performance Fee Account that will receive the performance fee collected by the Plasma Vault and later on distributed to IPOR DAO and recipients by FeeManager